docs/docs-components/data/components.js

```diff
@@ -383,6 +383,32 @@ const componentData: $ReadOnlyArray<ComponentData> = [
       },
     },
   },
+  {
+    id: 'ButtonLink',
+    platform: {
+      web: {
+        name: 'ButtonLink',
+        visual: {
+          svg: <Button />,
```

Comment:
can we  ask in #gestalt-team to make a BbuttonLink SVG?

Reply:
Asked, waiting for answer

Reply:
DONE